<launch>

  <group ns="manipulator1">
    <node pkg="manipulator" name="LMNodeName" type="LMNode.py"/>
  </group>

  <group ns="manipulator1">
    <node pkg="manipulator" name="IKNodeName" type="IKNode.py"/>
  </group>

  <group ns="manipulator1">
    <node pkg="manipulator" name="SKNodeName" type="SKNode.py"/>
  </group>

  <group ns="manipulator1">
    <node pkg="manipulator" name="VisualNodeName" type="VisualNode.py"/>
  </group>


  <!-- <group ns="turtlesim2">
    <node pkg="turtlesim" name="sim" type="turtlesim_node"/>
  </group>

  <node pkg="turtlesim" name="mimic" type="mimic">
    <remap from="input" to="turtlesim1/turtle1"/>
    <remap from="output" to="turtlesim2/turtle1"/>
  </node> -->

</launch>

